pkg/utils/common.go

```diff
@@ -31,6 +31,10 @@ import (
 	"go.goms.io/fleet/pkg/utils/informer"
 )
 
+const (
+	CustomResourceDefinitionKind = "CustomResourceDefinition"
+)
+
```

Comment:
please merge this with all the kind definition below

and clean up pkg/webhook/validation/uservalidation.go too